Source/Engine/Renderer/AmbientOcclusionPass.cpp

```diff
@@ -269,10 +273,9 @@ void AmbientOcclusionPass::Render(RenderContext& renderContext)
     // Request temporary buffers
     InitRTs(renderContext);
 
-    // Update and bind constant buffer
+    // Update and bind SSAO constant buffer
     UpdateCB(renderContext, context, 0);
-    context->BindCB(SSAO_CONSTANTS_BUFFER_SLOT, _shader->GetShader()->GetCB(SSAO_CONSTANTS_BUFFER_SLOT));
```

Comment:
Remove redundant call: `BindCB` is already called inside `UpdateCB` so this call is redundant.